File: website/docs/code-samples/rt-preview-oauth-setup.m
CrowdinProviderConfig *crowdinProviderConfig = [[CrowdinProviderConfig alloc]
    initWithHashString:@"{your_distribution_hash}"
    sourceLanguage:@"{source_language}"
    organizationName:@"{organization_name}"];

NSError *error;
CrowdinLoginConfig *loginConfig = [[CrowdinLoginConfig alloc]
    initWithClientId:@"{client_id}"
    clientSecret:@"{client_secret}"
    scope:@"project"
    error:&error];

if (!error) {
    CrowdinSDKConfig *config = [[[[CrowdinSDKConfig config]
        withCrowdinProviderConfig:crowdinProviderConfig]
        withRealtimeUpdatesEnabled:YES]
        withLoginConfig:loginConfig];

    [CrowdinSDK startWithConfig:config completion:^{
       // SDK is ready to use, put code to change language, etc. here
    }];
} else {
    NSLog(@"%@", error.localizedDescription);
    // CrowdinLoginConfig initialization error handling
}

File: website/docs/code-samples/rt-preview-oauth-setup.swift
let crowdinProviderConfig = CrowdinProviderConfig(hashString: "{your_distribution_hash}",
    sourceLanguage: "{source_language}",
    organizationName: "{organization_name}") // Optional. Required for Enterprise only

var loginConfig: CrowdinLoginConfig
do {
    loginConfig = try CrowdinLoginConfig(clientId: "{client_id}",
       clientSecret: "{client_secret}",
       scope: "project",
       redirectURI: "{redirectURI}")
} catch {
    print(error)
    // CrowdinLoginConfig initialization error handling
}

let crowdinSDKConfig = CrowdinSDKConfig.config()
    .with(crowdinProviderConfig: crowdinProviderConfig)
    .with(realtimeUpdatesEnabled: true)
    .with(loginConfig: loginConfig)
    .with(settingsEnabled: true)

CrowdinSDK.startWithConfig(crowdinSDKConfig, completion: {
    // SDK is ready to use, put code to change language, etc. here
})
